Source/CTPanoramaView.swift

```diff
@@ -31,6 +31,8 @@ import ImageIO
     
     @objc public var panSpeed = CGPoint(x: 0.005, y: 0.005)
     
+    @objc public var startAngleDegrees : Float = 0
```

Comment:
I think it's better to use radians here to be consistent since the `movementHandler(_:_:)` method also uses radians in its interface.

Reply:
Thank you for your response. I think you are right.